bees/twitterbee/twitterbeefactory.go

```diff
@@ -168,6 +168,11 @@ func (factory *TwitterBeeFactory) Events() []bees.EventDescriptor {
 					Description: "text content of the tweet",
 					Type:        "string",
 				},
+				{
+					Name:        "url",
+					Description: "URL of the tweet",
+					Type:        "string",
```

Comment:
Use the type "url" for those url parameters, please. I know it's not consistently used across all bees yet, but I'm about to make that switch in the near future.